src/lib/airtable/patches/patch-schema.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/src/lib/airtable/schema.js b/src/lib/airtable/schema.js
-index 7c751ee..e71d320 100644
```

Comment:
Not sure if this is necessary/what it's actually doing, but this was generated when I ran the git diff.